model-optimizer/mo/front/extractor.py

```diff
@@ -753,7 +753,7 @@ def add_outputs_identity(graph: Graph, outputs: list, add_edge: callable, params
     for output in outputs:
         fake_node_name = graph.unique_id(output)
         graph.add_node(fake_node_name, name=fake_node_name, identity=True, kind='op', op='Identity',
-                       infer=None, needs_removal=True)
+                       infer=None, needs_removal=True, symbol_dict={'op': 'Identity'})
```

Comment:
nit: There is an operation in MO made for such a purpose, called FakeOutput, I would suggest using it for consistency.

Reply:
FakeOutput cannot be used here because there is a transformation in MO that renames output nodes connected with FakeOutput nodes and sets output name to be tensor name.  And it is not what needed here, we need to keep original output names.